Apps/CesiumViewer/CesiumViewer.js

```diff
@@ -34,7 +34,7 @@ define([
      * 'debug'  : true/false,   // Full WebGL error reporting at substantial performance cost.
      * 'lookAt' : CZML id,      // The CZML ID of the object to track at startup.
      * 'source' : 'file.czml',  // The relative URL of the CZML file to load at startup.
-     * 'stats'  : true,         // Enable the FPS performance display.
+     * 'statistics' : true,     // Enable the FPS performance display.
```

Comment:
I don't think we should change `Viewer`, no need to create a breaking change for this now.